### ISSUE TITLE
Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

### ISSUE THREAD

**Issue by francisu** [kjur/jsrsasign#460]
I get this when running with node v12.16.2, jsrsasign v10.0.4

```
(node:1021) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
    at showFlaggedDeprecation (buffer.js:192:11)
    at new Buffer (buffer.js:279:3)
    at b64utoutf8 (/Users/francis/d/IdeaProjects/product/packages/server/.webpack/service/webpack:/Users/francis/d/IdeaProjects/product/.yarn/cache/jsrsasign-npm-10.0.4-d386c25616-19c95f2c20.zip/node_modules/jsrsasign/lib/jsrsasign.js:241:1197)
    at Function.module.exports.../../.yarn/cache/jsrsasign-npm-10.0.4-d386c25616-19c95f2c20.zip/node_modules/jsrsasign/lib/jsrsasign.js.KJUR.jws.JWS.parse (/Users/francis/d/IdeaProjects/product/packages/server/.webpack/service/webpack:/Users/francis/d/IdeaProjects/product/.yarn/cache/jsrsasign-npm-10.0.4-d386c25616-19c95f2c20.zip/node_modules/jsrsasign/lib/jsrsasign.js:251:3754)
```

**Comment by kjur** [comment]
Thank you for the report. I'll fix it in the future release.

**Comment by kjur** [comment]
This issue was fixed in the release 10.0.5 today.

**Comment by francisu** [comment]
Thanks for the fast fix!
